app/src/ble.c

```diff
@@ -222,6 +222,23 @@ int zmk_ble_clear_bonds() {
     return 0;
 };
 
+int zmk_ble_clear_all_bonds() {
```

Comment:
This is an issue with the existing functions too, but leaving the parameter list blank in C allows the function to accept any number of parameters of arbitrary types. The parameter list needs to be `(void)` for it to not take any parameters.
```suggestion
int zmk_ble_clear_all_bonds(void) {
```